client/src/api/index.js

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
+
+const API = axios.create({ baseURL: 'http://localhost:5000' });
```

Comment:
baseUrl shouldn't be hardcoded. We need to get it from .env through an app/config.js file or any other better modules.